src/inference/src/threading/ie_istreams_executor.cpp

```diff
@@ -409,6 +427,61 @@ void IStreamsExecutor::Config::UpdateHybridCustomThreads(Config& config) {
     }
 }
 
+IStreamsExecutor::Config IStreamsExecutor::Config::SetExecutorConfig(std::string name,
+                                                                     int num_streams,
+                                                                     ThreadBindingType thread_binding_type,
+                                                                     PreferredCoreType thread_core_type) {
+    Config streamExecutorConfig = Config{name, num_streams};
+    if (cpu_map_available()) {
+        if (name.find("CPU") == std::string::npos) {
+            streamExecutorConfig._logic_core_disable = true;
+            if (name.find("GPU") != std::string::npos) {
+                streamExecutorConfig._plugin_task = GPU_PRE_USED;
+            }
+        }
+        const auto proc_type_table = get_num_available_cpu_cores();
+        streamExecutorConfig._threads = streamExecutorConfig._streams;
+        streamExecutorConfig._threadBindingType = thread_binding_type;
+        streamExecutorConfig._threadPreferredCoreType = thread_core_type;
+        if (streamExecutorConfig._threadBindingType == CORES) {
+            streamExecutorConfig._bind_cores = true;
+        }
+        if (streamExecutorConfig._threadPreferredCoreType == LITTLE) {
+            streamExecutorConfig._small_core_streams = std::min(num_streams, proc_type_table[0][EFFICIENT_CORE_PROC]);
+            streamExecutorConfig._threads_per_stream_small = 1;
+        } else {
+            streamExecutorConfig._big_core_streams = std::min(num_streams, proc_type_table[0][MAIN_CORE_PROC]);
+            streamExecutorConfig._threads_per_stream_big = 1;
+        }
+        if (streamExecutorConfig._bind_cores) {
+            auto core_type =
+                streamExecutorConfig._threadPreferredCoreType == LITTLE ? EFFICIENT_CORE_PROC : MAIN_CORE_PROC;
```

Comment:
Still need `ANY` core type for latency mode stream which include both Pcore and Ecore